wiki/templates/wiki/dir.html

```diff
@@ -22,14 +22,14 @@
       {% trans "Add article" %}
     </a>
   </div>
-  <div class="pull-right">
-    {{ filter_form.query }}
+  <div class="pull-right input-prepend">
```

Comment:
Add ?w=1 to suppress whitespace differences (though note that then you won't see comments or be able to create them).